test/local/metrics_tests.js

```diff
@@ -207,13 +205,7 @@ DB.connect(config)
             }
           }
         }
-        Object.keys(mocks).forEach(function (key) {
```

Comment:
There was some trickery involved in getting the old coverage tool and proxyquire working. seems like it works fine everywhere now